source/Parser.cpp

```diff
@@ -341,13 +341,22 @@ MemberSyntax* Parser::parseMember() {
             // TODO: Don't assume we have an assert here; this could be an accidental label or something
             auto name = consume();
             auto label = alloc.emplace<NamedLabelSyntax>(name, expect(TokenKind::Colon));
-            return alloc.emplace<ConcurrentAssertionMemberSyntax>(attributes, parseConcurrentAssertion(label, nullptr));
+            auto statement = parseAssertionStatement(label, nullptr);
+            if (statement->kind == SyntaxKind::ImmediateAssertStatement) {
```

Comment:
There are a few kinds of things covered here right? Covers and Assumes in addition to Asserts.